storage/replica.go

```diff
@@ -594,10 +592,10 @@ func (r *Replica) addAdminCmd(ctx context.Context, args proto.Request) (proto.Re
 
 	switch tArgs := args.(type) {
 	case *proto.AdminSplitRequest:
-		resp, err := r.AdminSplit(*tArgs)
+		resp, err := r.AdminSplit(*tArgs, r.Desc())
```

Comment:
Unless you think it's better just to put the RangeDescriptor into the AdminSplitRequest proto.